Ukhu/Banka
# #165173527 Build POST /transactions/<accountNumber>/credit endpoint

#### What does this PR do?
Test and build the API endpoint for crediting a user's bank account

#### Description of Task to be completed?
1. Write unit tests for the route
2. Set up the route
3. Add the necessary middlewares and handlers
4. Add server-side validations
5. Ensure test coverage is sufficient enough

#### How should this be manually tested?
1. Clone the repo
2. Run 'npm run test'
3. Start the server with the command 'npm run dev'
4. Start by sending a post request on Postman to the signup route to create a new user
     - Also create an Staff(account) via the signup endpoint
     - Create a user bank account via the accounts endpoint
5. Test on Postman using the following fields in the request body:
    'type' - the type of transaction ( 'credit' in this case)
    'accountNumber' - account number of the account to be credited
    'token' - the cashier's token created from the response on the signup route
6.  The response gotten should be a JSON object with a status property and a data or error(s) property for success and failure respectively.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#165173527

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
